platform/public_api/apps/index.json

```diff
@@ -391,44 +393,6 @@
       "description": "App UUIDv4",
       "required": true,
       "type": "string"
-    },
-    "appName": {
```

Comment:
This and later parameters should removed in this version, ot were they removed in 7.0?

Reply:
these are unused parameters, they were unused also in 6.1